example.env

```diff
@@ -1,10 +1,14 @@
 ENVIRONMENT=dev
 DOMAIN=127.0.0.1
 
-SECRET_KEY=secret
-CSRF_SECRET=secret
-SESSION_SECRET_KEY=secret
+REFRESH_TOKEN_EXPIRE_MINUTES=1440
 ACCESS_TOKEN_EXPIRE_MINUTES=30
+VERIFY_TOKEN_SECRET_KEY=secret1
+ACCESS_TOKEN_SECRET_KEY=secret2
+REFRESH_TOKEN_SECRET_KEY=secret3
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Example environment file contains hard-coded secrets.

It's recommended to remove or replace hard-coded secrets with placeholders to prevent accidental exposure.